website/docs/reference/widgets/datepicker.md

```diff
@@ -1,132 +1,168 @@
-| [**Height**](/reference/widgets/#height)        | It configures how a widget’s height reacts to content changes. It has three possible configurations:<br/>**Fixed**: The height of the widget remains as set using drag and resize.<br/>**Auto Height**: The height of the widget reacts to content changes.<br/>  **Auto Height with limits**: Same as Auto height, with a configurable option to set the minimum and maximum number of rows that can be occupied by the widget.                                      |
+* ISO 8601
+* LLL
+* LL
+* YYYY-MM-DD HH:mm
+* YYYY-MM-DDTHH:mm:ss
+* YYYY-MM-DD hh:mm:ss A
+* DD/MM/YYYY HH:mm
+* D MMMM, YYYY
+* H:mm A D MMMM, YYYY
+* YYYY-MM-DD
+* MM-DD-YYYY
+* DD-MM-YYYY
+* MM/DD/YYYY
+* DD/MM/YYYY
+* DD/MM/YY
 
 
-### Binding Properties
+If you need more advanced date formatting options, you can use the [**Moment.js**](https://momentjs.com/docs/) library which is installed by default in Appsmith. Moment.js is a JavaScript library for parsing, validating, manipulating, and formatting dates. With Moment.js, you can define your own custom date formats using a simple syntax.
 
-These properties allow you to bind your Datepicker widget with any other widget in queries or JS objects. The following table lists all the binding properties.
+To use Moment.js with Appsmith, you can simply write JavaScript code in the widget's **Date Format** property. For example, you can use the following code to set the default date value to **today's date** in a specific format:
 
-| Binding Property  | Description                                                                                                                                                    |
-| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **formattedDate** | Contains the date value currently selected within the Datepicker widget. This value changes if the default value is updated or if the user inputs a new value. |
-| **selectedDate**  | This is the ISO date string selected in the Datepicker widget. This value changes if the default value is updated or if the user inputs a new value.           |
-| **isDisabled**    | Reflects the state of the widget's **Disabled** setting _(bool)_.                                                                                              |
-| **isVisible**     | Reflects the state of the widget's **Visible** setting _(bool)_.                                                                                               |
+```JS
+{{moment().format('YYYY-MM-DD')}}
```

Comment:
In the Date format field I think you only need to specify the format in single quotes inside mustache syntax. Check this with the App viewers pod